Tasks/JenkinsQueueJobV2/job.ts

```diff
@@ -276,7 +276,9 @@ export class Job {
                 if (thisJob.queue.TaskOptions.capturePipeline) {
                     const downstreamProjects = thisJob.Search.ParsedTaskBody.downstreamProjects || [];
                     downstreamProjects.forEach((project) => {
-                        new Job(thisJob.queue, thisJob, project.url, null, -1, project.name); // will add a new child to the tree
+                        if (project.color !== 'disabled') {
```

Comment:
The 'color' field is how Jenkins identifies the status of a project. This field will be set to disabled if the project has been disabled.

Reply:
So, we only add jobs to the tree if they are not disabled.  What about the root job -- can it be disabled?

Reply:
Attempting to queue a disabled root job results in a 404 from Jenkins, which causes our build to fail right away.